e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -88,8 +88,9 @@ Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage
   I.seeAnswerInTab(5, 'Hearing 2', '', hearingDetails[1].type.somethingElse);
   I.seeAnswerInTab(6, 'Hearing 2', 'Give details', hearingDetails[1].giveDetails);
   I.seeAnswerInTab(1, 'Judge and legal advisor', 'Judge or magistrate\'s title', hearingDetails[1].judgeAndLegalAdvisor.judgeTitle);
```

Comment:
(0) If there was a way to set 'Judge or magistrate's title' in the tab to be otherTitle if we have one, that would be great, but I don't think it's possible - another option would be to hide this field if 'Other' is selected, but this would also happen during input. I don't think we can make this look any better in the tab, at least not right now ⏳

Reply:
Yeah not right now :(,not much options we currently have.